src/exercises/adapt-c/README.md

```diff
@@ -1 +1,54 @@
 # Adapt a C Program to CHERI C
+
+This excercise presents an example C program that includes capability-related
+issues that might appear as bugs in software initially developed for non-CHERI
+architectures.  The example C program is `cat(1)` from CheriBSD (and hence
+FreeBSD) modified to introduce the issues that we want to investigate.
+
+
+1. Read Sections 4.2, 4.2.1, 4.2.3 from the
+   [CHERI C/C++ Programming Guide](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf).
+   In Section 4.2.1, read only information on the following C-language types:
+   `long`, `uintptr_t` and `char *,...` (pointer types).
```

Comment:
That's probably fine for the book itself; are you going to present the relevant material in the pre-exercise slideware?

Reply:
Yes, the sections are quite short but I will summarise them in the slides.